lib/portlayer/exec/handle.go

```diff
@@ -179,8 +192,6 @@ func (h *Handle) Create(ctx context.Context, sess *session.Session, config *Cont
 
 	// update the handle with Metadata
 	h.ExecConfig = config.Metadata
-	// add create time to config
```

Comment:
It moved to the createHandler -- so when the config is created it's populated there